plugins/gatsby-plugin-generate-doc-json/gatsby-node.js

```diff
@@ -0,0 +1,40 @@
+const fs = require('fs');
+const path = require('path');
+const unified = require('unified');
+const toHast = require('mdast-util-to-hast');
+const html = require('rehype-stringify');
+const removeImports = require('remark-mdx-remove-imports');
+const removeExports = require('remark-mdx-remove-exports');
+
+const htmlGenerator = unified().use(removeImports).use(removeExports).use(html);
+
+exports.onPostBuild = async ({ graphql, store }) => {
+  const { program } = store.getState();
+
+  const query = `query {
+    allMdx(filter: {fileAbsolutePath: {regex: "/src/content/docs/"}}) {
+      nodes {
+        mdxAST
+        slug
```

Comment:
Interesting! I didn't realize there was a `slug` field already here. It looks like we are using [`fields.slug`](https://github.com/newrelic/docs-website/blob/develop/plugins/gatsby-plugin-release-note-rss/gatsby-node.js#L30-L32) elsewhere. Are these 2 equivalent?